.docker/php81_build_phar

```diff
@@ -0,0 +1,19 @@
+FROM php:8.1-cli-alpine as build-stage
```

Comment:
image is not used atm, was a by-product when trying to build a release: this builds a docker image for "box" from scratch, i.e. box is build within docker and then copied over into the final image.

could be removed, if not of interest.

Reply:
Yes please!